grpc/config.go

```diff
@@ -62,10 +62,14 @@ func (c *Config) InitDefaults() error { //nolint:gocognit
 		return errors.E(op, errors.Errorf("mailformed grpc address, provided: %s", c.Listen))
 	}
 
-	if c.Proto != "" {
-		if _, err := os.Stat(c.Proto); err != nil {
+	for _, pf := range c.Proto {
```

Comment:
Do not use `range` for the slices to avoid copying. Use a regular loop -> `for i:=0; i<len; i++`.

Reply:
What a marvelous premature optimization